handler.go

```diff
@@ -304,7 +302,7 @@ func (handler *Handler) handlePacket(packet packets.Packet, direction packets.Di
 
 func (handler *Handler) handleProxy() (err error) {
 	if handler.Handshake.Protocol != packets.ProtocolVersion {
-		kick := packets.NewLoginKick(&packets.ChatMessage{Text: "Server version: 1.14.4"}) // ta wersja nie powinna byc tu na sztywno
+		kick := packets.NewLoginKick(&packets.ChatMessage{Text: "Server version:" + packets.GameVersion.Name}) // ta wersja nie powinna byc tu na sztywno
```

Comment:
No to już nie jest, komentarz do wywalenia 😛

Reply:
Ogólnie to i można mergować jak dla mnie, z tych error readerów to trzeba issue zrobić i większy refactoring po prostu.